target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_arcadyan_vgv952cjw33-e-ir.dtsi

```diff
@@ -0,0 +1,419 @@
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mips/lantiq_rcu_gphy.h>
+
+
+/ {
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <100>;
+		rfkill {
+			label = "wps";
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+		reset {
+			label = "reset";
+			gpios = <&gpio 40 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	usb_vbus: regulator-usb-vbus {
+		compatible = "regulator-fixed";
+
+		regulator-name = "USB_VBUS";
+
+		regulator-min-microvolt = <5000000>;
+		regulator-max-microvolt = <5000000>;
+
+		gpio = <&gpio 33 GPIO_ACTIVE_HIGH>;
+		enable-active-high;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power_green: power {
+		label = "VGV952CJW33:red:power";
+		gpios = <&gpio 31 GPIO_ACTIVE_HIGH>;
+		default-state = "keep";
+		};
+	};
+
+	spi {
```

Comment:
Please name this spi-gpio or gpio_spi.

Reply:
yes i do but what are the meaning or what are the consequenses ?

Reply:
Typically the flash is labelled spi and thus the &spi label is somewhat reserved. Though this is just a node name and causes no functional collision, I would prefer the suggested more specific name to prevent confusion.